de/linearModel.py

```diff
@@ -0,0 +1,13 @@
+from sklearn.linear_model import Lasso
+import numpy as np
+
+def runFitting(data, U=None, max_iter=300000):
```

Comment:
Probably want an option here to pass in the regularization strength, as we'll want to adjust that in the future.